srcpkgs/nagios/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'nagios'
+pkgname=nagios
+version=4.4.6
+revision=1
+build_style=gnu-configure
+
+_nagios_user="_nagios"
+_nagios_group="_nagios"
+_instdir="/usr/share/nagios"
+_bindir="/usr/bin"
+_vardir="/var/nagios"
+_confdir="/etc/nagios"
+_httpdconfdir="/etc/webapps/$pkgname/"
+_checkresultdir="/var/nagios/spool/checkresults"
+_cgibindir="/usr/libexec/nagios/cgi"
+
+configure_args="
+	--with-nagios-user=$_nagios_user
+	--with-nagios-group=$_nagios_group
+	--prefix=$_instdir
+	--bindir=$_bindir
+	--localstatedir=$_vardir
+	--sysconfdir=$_confdir
+	--with-httpd-conf=$_httpdconfdir
+	--with-checkresultdir=$_checkresultdir
+	--with-cgibindir=$_cgibindir
+	--enable-embedded-perl"
+make_build_args="all"
+make_install_args="
+	prefix=$_instdir
+	BINDIR=$_bindir
+	LOGDIR=$_vardir
+	CFGDIR=$_confdir
+	HTTPD_CONF=$_httpdconfdir
+	CHECKRESULTDIR=$_checkresultdir
+	CGIDIR=$_cgibindir
+	install install-config"
+
+makedepends="perl unzip"
+make_dirs="/etc/nagios 0775 _nagios _nagios"
+conf_files="/etc/nagios/nagios.cfg"
+
+short_desc="Nagios Core"
```

Comment:
"host/service/network monitoring program" or similar, "Nagios Core" isn't descriptive.

Reply:
will change

Reply:
not changed

Reply:
It is changed.

It's in a separate commit pushed 10 days ago to this same branch.